work_queue/src/bindings/python3/work_queue.binding.py

```diff
@@ -1745,19 +1746,30 @@ def wait(self, timeout=WORK_QUEUE_WAITFORTASK):
     # @param self       Reference to the current work queue object.
     # @param fn         The function that will be called on each element
     # @param seq        The sequence that will call the function
-    def map(self, fn, seq):
-        results = [None] * len(seq)
+    # @param chunk_size The number of elements to process at once
+
+    def Nmap(self, fn, array, chunk_size=1):
```

Comment:
Call this one `map` since it's a drop in replacement for the old one.

Reply:
Op yes sorry I meant to but forgot to delete the N.